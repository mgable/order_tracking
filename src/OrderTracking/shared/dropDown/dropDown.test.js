import React from 'react';
import ReactDOM from 'react-dom';
import { shallow, render } from 'enzyme';
import { DropDown } from './';

var props = {onSetSelected: jest.fn(), items: [], label: "foo", id: "bar"}

describe("Dropdown", () => {
	it('renders without crashing', () => {
		const wrapper = render(<DropDown props={ {...props} } />);
		expect(wrapper).toMatchSnapshot();
	});
});
