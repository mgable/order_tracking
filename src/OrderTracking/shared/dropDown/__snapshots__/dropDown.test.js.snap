// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dropdown renders without crashing 1`] = `
<div
  class="dropdown"
>
  <button
    class="btn btn-primary"
    type="button"
  >
    foo
  </button>
</div>
`;
